src/components/menu/menu.stories.svelte

```diff
@@ -0,0 +1,63 @@
+<script lang="ts">
+  import { Meta, Story, Template } from '@storybook/addon-svelte-csf'
+
+  import Menu from './menu.svelte'
+  import Icon from '../icon/icon.svelte'
+  import { cssProperties, sizes } from '../formItem/formItem.svelte'
+  import SlotInfo from '../../storyHelpers/SlotInfo.svelte'
+  import Slot from '../../storyHelpers/Slot.svelte'
+  import { modes } from '../formItem/formItem.svelte'
+
+  const countries = {
+    'nz': 'New Zealand',
+    'au': 'Australia',
+    'gb': 'United Kingdom',
+    'ca': 'Canada'
+  }
+</script>
+
+<Meta
+  title="Menu"
+  component={Menu}
+  argTypes={{
+    ...cssProperties,
+    placeholder: {
+      defaultValue: 'select...'
+    },
+    label: {
+      defaultValue: 'Label'
+    },
+    size: {
+      control: 'select',
+      options: sizes
+    },
+    mode: {
+      control: 'select',
+      options: modes
+    }
+  }}
+/>
+
+<Template let:args>
+  <div class="container">
+    <Menu value={undefined} {...args}>
+      <div slot="anchor">
+        <Icon name="more-horizontal" />
+      </div>
+      <div slot="label">{args.label}</div>
```

Comment:
I don't think this component needs a label, or if it does, it should be part of the anchor

Reply:
makes sense. I've removed this for now. However, we might need this in the future. ex:

<img width="273" alt="image" src="https://github.com/brave/leo/assets/8665427/69c4d0a9-25bc-441e-ba84-a1f92d5fecef">

Reply:
Do you mean the "Available language models" part? Would that just be a non-selectable item in the menu, provided in the default slot?